OpenRA.Mods.Common/Activities/HarvesterDockSequence.cs

```diff
@@ -89,6 +100,7 @@ public override bool Tick(Actor self)
 				case DockingState.Complete:
 					Harv.LastLinkedProc = Harv.LinkedProc;
 					Harv.LinkProc(null);
+					NotifyUndocked(self);
```

Comment:
Is there a reason this is here and not on `DockingState.Undock`, together with `OnStateUndock(self);`, to match `NotifyDocked`?

Reply:
We currently call notifications before dock animation started and after dock animation has completed. Not when dock tick starts and ends. It's something I'd like to change in a future PR